lists/global.csv

```diff
@@ -21,6 +21,7 @@ http://anonymouse.org/,ANON,Anonymization and circumvention tools,2014-04-15,cit
 http://archive.org/,HOST,Hosting and Blogging Platforms,2014-04-15,citizenlab,Updated by OONI on 2017-02-14
 https://archive.org/web/,MMED,Media sharing,2014-04-15,citizenlab,Updated by OONI on 2017-02-14
 https://archiveofourown.org/,CULTR,Culture,2017-10-25,sinarproject,
+https://www.asstr.org/,CULTR,Culture,2019-10-17,sinarproject,erotic fiction
```

Comment:
This site doesn't seem to work (perhaps the site is hosted on an unreliable server) -- I'd propose removing it (unless if we can replace it with a different URL?).

Reply:
Works for me. Are you using CF 1.1.1.1 DNS by any chance?

Reply:
Hm, I'm able to access it now.